### ISSUE TITLE
[Idea/suggestion] Command/gamemode: /creativeblocks <player>

### ISSUE THREAD

**Issue by kimfy** [squeek502/CreativeBlocks#11]
`/creativeblocks <player>` would put the specified player in the creativeblocks gamemode. This is intended for servers where not everyone wants the mod enabled. So basically what I'm asking here is that players that have `/gamemode creativeblocks` enabled would be able to use this as intended while not affecting everyone else. The blocks placed by those in `/gamemode creativeblocks` would be breakable for people not in the gamemode, however, they wouldn't produce a drop.

I thought of this idea when designing new gamemodes for my server like `/gamemode magic`, `/gamemode tec`h. `/gamemode magic` would force you to only use magic mods and the same for `/gamemode tech`.

If it cannot be done then, oh well. Thank you squeek502, I love this mod to no end and would like to enjoy it on my server as well without affecting others.

**Comment by squeek502** [comment]
Definitely possible. Just note that the people with creative blocks could then give infinite amount of whitelisted blocks to any player (creative-blocks-enabled player places a block, non-creative-blocks-enabled player breaks it).

**Comment by kimfy** [comment]
Oh, so you can't prevent it from being dropped for those who aren't in `/gamemode creativeblocks`?

**Comment by squeek502** [comment]
I could, but then non-creative-blocks-enabled players would be completely unable to obtain that block from the world.

**Comment by squeek502** [comment]
Oh, I misread your original suggestion. Marking certain placed blocks as 'creative' (and then make non-'creative' placed blocks of the same type behave normally for everyone) _might_ be possible, but that would be a separate feature.

Also, I'm not sure when I'll have time to work on this mod. Might not be for a little while.

**Comment by kimfy** [comment]
Yeah that was what I had in mind. Take your time, no ones in a rush. :-)
